test/e2e/gameserver_test.go

```diff
@@ -1364,93 +1364,93 @@ func TestCounters(t *testing.T) {
 	}{
 		"GetCounterCount": {
 			msg:  "GET_COUNTER_COUNT games",
-			want: "1",
+			want: "COUNTER: 1\n",
```

Comment:
To make this fun - you'll need to wait until the new version of simple-game-server is in production before changing the e2e test.

Which also means this PR will need an increment to the example image version 😃 up to you if you want to do it in separate PR's or not.

But otherwise, this looks great 👍🏻

Reply:
Could you share your preference? I'll go with that🙂

Reply:
During the testing of some modified methods, I noticed that the changes aren't reflecting properly. I will draft, test, and update the code accordingly.

Reply:
> Could you share your preference? I'll go with that🙂

No preference 😃